apps/agreement/contracts/Agreement.sol

```diff
@@ -1259,8 +1258,9 @@ contract Agreement is IArbitrable, ILockManager, IAgreement, IACLOracle, AragonA
     * @return Identification number of the current agreement setting
     */
     function _getCurrentSettingId() internal view returns (uint256) {
-        // Note: technically this is not true if you have an uninitialized proxy and call the public getCurrentSettingId()
-        return nextSettingId - 1; // an initial setting is created during initialization, thus length will be always greater than 0
+        // An initial setting is created during initialization, thus length will be always greater than 0
+        // Note that all the places where this helper is being used, an initialization check is previously performed
```

Comment:
Let's place this note in the `@dev` comments. Worst case, we can also just check for 0.